pages/common/fakedata.md

```diff
@@ -0,0 +1,24 @@
+# fakedata
+
+> CLI utility for fake data generation.
+> More information: <https://github.com/lucapette/fakedata>.
+
+- List generators (all kinds of data that can ge generated):
```

Comment:
```suggestion
- List generators (all kinds of data that can be generated):
```

Reply:
Should I amend the commit (push --force) ? Or commit the suggestion (new commit)?

Reply:
The latter.  No need to keep one commit because we auto squash before merging.